src/settings/toml/manifest.rs

```diff
@@ -239,6 +241,9 @@ impl Manifest {
 
             // don't inherit vars
             target.vars = environment.vars.clone();
+
+            // don't inherit blobs
+            target.blobs = environment.blobs.clone();
```

Comment:
i don't remember why we chose not to inherit vars... 🤔

Reply:
i think these should likely be inherited; cc @rita3ko any thoughts?

Reply:
these should be inherited i think.

Reply:
My only reason for not inheriting these, is it feels weird to do something different than for vars.

Reply:
@ashleymichal @rita3ko Do y'all have a final opinion here? Y'all have more context than I do about what should/should not get inherited